hubspot/api.py

```diff
@@ -182,7 +182,10 @@ def get_sync_errors(limit=200, offset=0):
     while len(errors) > 0:
         yield from errors
         offset += limit
-        errors = paged_sync_errors(limit, offset)
+        try:
+            errors = paged_sync_errors(limit, offset)
+        except StopIteration:
```

Comment:
Does `paged_sync_errors` actually throw a `StopIteration` here sometimes? I'd expect it to maybe throw an `HTTPError`.

Reply:
it was throwing StopIteration and test cases were failing because of it that's why I have added that.

Reply:
👍